Description: Various config.test fixes.
             Build ibase sql plugin against firebird.
             Add gstreamer include path.
Author: Fathi Boudra <fabo@debian.org>
Last-Update: 2010-05-24

Index: qt-everywhere-opensource-src-4.7.1/config.tests/unix/ibase/ibase.pro
===================================================================
--- qt-everywhere-opensource-src-4.7.1.orig/config.tests/unix/ibase/ibase.pro	2010-11-15 17:22:59.574553082 +0000
+++ qt-everywhere-opensource-src-4.7.1/config.tests/unix/ibase/ibase.pro	2010-11-15 17:23:08.798553081 +0000
@@ -1,4 +1,4 @@
 SOURCES = ibase.cpp
 CONFIG -= qt dylib
 mac:CONFIG -= app_bundle
-LIBS += -lgds
+LIBS += -lfbclient
Index: qt-everywhere-opensource-src-4.7.1/config.tests/unix/sqlite/sqlite.pro
===================================================================
--- qt-everywhere-opensource-src-4.7.1.orig/config.tests/unix/sqlite/sqlite.pro	2010-11-15 17:22:59.606553082 +0000
+++ qt-everywhere-opensource-src-4.7.1/config.tests/unix/sqlite/sqlite.pro	2010-11-15 17:23:08.798553081 +0000
@@ -1,3 +1,4 @@
 SOURCES = sqlite.cpp
 CONFIG -= qt dylib
 mac:CONFIG -= app_bundle
+LIBS += -ldl
Index: qt-everywhere-opensource-src-4.7.1/config.tests/unix/gstreamer/gstreamer.pro
===================================================================
--- qt-everywhere-opensource-src-4.7.1.orig/config.tests/unix/gstreamer/gstreamer.pro	2010-11-15 17:22:59.586553082 +0000
+++ qt-everywhere-opensource-src-4.7.1/config.tests/unix/gstreamer/gstreamer.pro	2010-11-15 17:23:08.802553081 +0000
@@ -1,3 +1,4 @@
 SOURCES = gstreamer.cpp
 CONFIG -= qt
 LIBS += -lgstinterfaces-0.10 -lgstvideo-0.10 -lgstbase-0.10
+INCLUDEPATH += /usr/include/gstreamer-0.10
